examples/connext_dds/compression/c++/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# (c) 2018 Copyright, Real-Time Innovations, Inc. All rights reserved.
```

Comment:
This file was not included in the `rticonnextdds-examples/examples/connext_dds/CMakeLists.txt` file